### PR TITLE
Update: use jenkins built-in variable to get job url

### PR DESCRIPTION
**Pull request by zdtsw**
Easy to skip setting of jenkins_url from parameters and job can point to the running one.
not always use https://ci.adoptium.net/view/Tooling/job/nightlyBuildAndTestStats_*

### CODE REVIEW

**Review by karianna — APPROVED** [review]

**Review by gdams — APPROVED** [review]
LGTM (I was thinking this when I looked at the job) but I'd like to get @andrew-m-leonard's review before we merge

**Review by andrew-m-leonard — APPROVED** [review]

**Pull request merged**
